amethyst_renderer/src/error.rs

```diff
@@ -1,126 +1,70 @@
 //! Renderer error types.
 
-use std::error::Error as StdError;
-use std::fmt::{Display, Formatter};
-use std::fmt::Result as FmtResult;
-use std::result::Result as StdResult;
-
 use gfx;
 use gfx_core;
```

Comment:
This is not recommended error_chain usage. It is advised to use paths from root, i.e `::gfx_core::*`

Reply:
However, if it works I don't see any problems.